code/modules/projectiles/ammunition/energy_lens.dm

```diff
@@ -42,14 +42,14 @@
 	fire_sound = 'sound/weapons/lasercannonfire.ogg'
 
 /obj/item/ammo_casing/energy/laser/pulse
-	projectile_type = /obj/item/projectile/beam/pulse
+	projectile_type = /obj/item/projectile/beam/pulse/hitscan
 	muzzle_flash_color = LIGHT_COLOR_DARKBLUE
 	e_cost = 200
 	select_name = "DESTROY"
 	fire_sound = 'sound/weapons/pulse.ogg'
 
 /obj/item/ammo_casing/energy/laser/scatter/pulse
-	projectile_type = /obj/item/projectile/beam/pulse
+	projectile_type = /obj/item/projectile/beam/pulse/hitscan
```

Comment:
How does this work with scattering? Does it properly shoot out 5 hitscan beams in multiple directions? If it doesn't work, it should probably left as normal pulse rounds.

Reply:
It works fine